src/parser/jobs/war/modules/index.ts

```diff
@@ -1,15 +1,15 @@
 import {ActionTimeline} from './ActionTimeline'
+import {AoE} from './AoE'
 import {Gauge} from './Gauge'
-import InnerRelease from './InnerRelease'
-import MultiHitSkills from './MultiHitSkills'
-import OGCDDowntime from './OGCDDowntime'
-import StormsEye from './StormsEye'
+import {InnerRelease} from './InnerRelease'
+import {OGCDDowntime} from './OGCDDowntime'
+import {StormsEye} from './StormsEye'
 
 export default [
 	ActionTimeline,
 	Gauge,
 	InnerRelease,
-	MultiHitSkills,
+	AoE,
```

Comment:
Nit for the sake of leaving a comment other than "looks good to me"; you should move this one up above `Gauge` so it's alphabetical/in the same order as the imports.